core/app/serializers/uffizzi_core/api/cli/v1/project_serializer.rb

```diff
@@ -16,7 +16,10 @@ def default_compose
   end
 
   def deployments
-    object.deployments.existed
+    object.deployments.active.map do |deployment|
+      deployment.state = UffizziCore::DeploymentService.failed?(deployment) ? 'failed' : 'active'
```

Comment:
Instead of `failed` and `active` let's use constant like this `UffizziCore::Deployment::STATE_FAILED`